#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_build_system.dpatch by David Paleino <d.paleino@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Transition to Mono 2.0

@DPATCH@
diff -urNad beagle-0.3.8~/configure.in beagle-0.3.8/configure.in
--- beagle-0.3.8~/configure.in	2008-06-12 18:06:25.000000000 +0200
+++ beagle-0.3.8/configure.in	2008-11-28 04:18:18.000000000 +0100
@@ -54,14 +54,14 @@
 	AC_MSG_ERROR([You need to install the Mono runtime])
 fi
 
-AC_PATH_PROG(MCS, gmcs, no)
+AC_PATH_PROG(MCS, mono-csc, no)
 if test "x$MCS" = "xno"; then
-	AC_MSG_ERROR([You need to install the Mono gmcs compiler])
+	AC_MSG_ERROR([You need to install the Mono csc compiler])
 fi
 
 AC_MSG_CHECKING([for mono.pc])
 if test -z `$PKG_CONFIG --variable=prefix mono`; then
-  AC_MSG_ERROR([missing the mono.pc file, usually found in the mono-devel package])
+  AC_MSG_ERROR([missing the mono.pc file, usually found in the libmono-dev package])
 else
   AC_MSG_RESULT([found])
 fi
